src/lib/download.js

```diff
@@ -18,10 +18,20 @@ export default (filename) => {
       stream: process.stdout
     }, progress.Presets.rect)
 
+    if (attempts === 0) {
+      reject(new Error(clk.red(`Unable to download ${fullUrl}`)))
+      return
+    }
+
     http.get(
       fullUrl,
       (response) => {
         let cur = 0
+        const responseType = response.headers['content-type']
+        if (!responseType.includes('audio')) {
+          console.log((clk.red(`No audio file found: ${clk.magenta(fullUrl)} \ntrying again in ${constants.RETRY_INTERVAL / 60} minutes.`)))
```

Comment:
[minor] we already have moment.js for data/time manipulation, I think we can use that instead of calculating the minutes (which gives a floating point number that we don't need)